cmd/setup-etcd-environment/run.go

```diff
@@ -193,7 +193,8 @@ func runRunCmd(cmd *cobra.Command, args []string) error {
 
 func (s *SetupEnv) getExportEnv() (map[string]string, error) {
 	// if etcd is managed by the operator populate ENV from configmap
-	if s.isEtcdScaling() {
+	//if s.isEtcdScaling() {
```

Comment:
do you want to add a note here in comments since this is a temporary fix?

Reply:
also // needs a space after

Reply:
done, thanks!